vite.config.ts

```diff
@@ -0,0 +1,27 @@
+import react from "@vitejs/plugin-react";
+import {defineConfig} from "vite";
+
+
+// https://vite.dev/config/
+export default defineConfig({
+    appType: "mpa",
+    base: "./",
+    build: {
+        assetsDir: "",
```

Comment:
Put all assets under `dist` instead of `dist/assets`, which is more convenient for deployment (some customer's s3 webui doesn't upload files in a folder when it's DND)